sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/CommunicationTokenCredential.java

```diff
@@ -61,19 +62,14 @@ public CommunicationTokenCredential(TokenRefresher tokenRefresher) {
      * CallbackOffsetMinutes defaulted to two minutes. To modify this default, call
      * setCallbackOffsetMinutes after construction
      * 
-     * @param tokenRefresher implementation to supply fresh token when reqested
+     * @param tokenRefreshOptions implementation to supply fresh token when reqested
      * @param initialToken serialized JWT token
-     * @param refreshProactively when set to true, turn on proactive fetching to call
-     *                           tokenRefresher before token expiry by minutes set
-     *                           with setCallbackOffsetMinutes or default value of
-     *                           two minutes
      */
-    public CommunicationTokenCredential(TokenRefresher tokenRefresher, String initialToken,
-            boolean refreshProactively) {
-        this(tokenRefresher);
+    public CommunicationTokenCredential(TokenRefreshOptions tokenRefreshOptions, String initialToken) {
```

Comment:
I think we should rename `initialToken` to simply `token` everywhere.

For the constructor with refresh options, `token` can be an optional argument in the options bag, we don't need it as a top-level arg in the constructor.

Reply:
Sure, Java does not have optional arguments so I've overloaded the constructor with the token param